packages/core/src/shapes/Shapes.ts

```diff
@@ -70,6 +72,7 @@ export const ShapeDef = (shapedef: {
     bbox: shapedef.bbox,
     isLinelike: shapedef.isLinelike || false,
     isRectlike: shapedef.isRectlike || false,
+    isPolygonlike: shapedef.isRectlike || false,
```

Comment:
Is this intentional, i.e. all rect like shapes are also polygon like?

Reply:
Nice catch, it is a typo. Thanks!